Dockerfile_build

```diff
@@ -11,6 +11,7 @@ WORKDIR /usr/src/app
 
 
 RUN apk --no-cache add zip
+RUN npm update -g npm
```

Comment:
Generalnie, zwykle zaleca się ustawiać konkretne wersje paczek w dockerfile, bo inaczej dziwnie się zachowuje w keszem (czasem w keszu będzie stare, czasem ściągnie nowe). W przypadku zipa to tam pewnie mały problem, ale już różne wersje npma to chyba trochę ryzykowne...

Reply:
(nie wiem, czy kojarzysz, jak keszuje docker, ale mogę wyjaśnić jak coś)